policy/serialization.go

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2017 Kamil Pawlowski, Ignasi Barrera
```

Comment:
This should only contain your name until I add something.  We're  going to need a contributors file. At which point I think this goes away.